WooCommerce/Classes/POS/Controllers/PointOfSaleItemsController.swift

```diff
@@ -50,6 +59,12 @@ protocol PointOfSaleItemsControllerProtocol {
         await loadFirstPage(base: base)
     }
 
+    @MainActor
+    func searchItems(searchTerm: String, baseItem: ItemListBaseItem) async {
+        itemProvider.fetchStrategy = itemFetchStrategyFactory.searchStrategy(searchTerm: searchTerm)
```

Comment:
Should we set the strategy back to `default` somewhere?

Reply:
I figured we could add that if we need it... but I wasn't quite sure if it would be needed in practice, so left it for now.